draft-ietf-quic-tls.md

```diff
@@ -618,11 +618,16 @@ application data, a server MUST treat receiving early data on the TLS connection
 as a connection error of type PROTOCOL_VIOLATION.
 
 
-## Rejecting 0-RTT
+## Accepting and rejecting 0-RTT
 
-A server rejects 0-RTT by rejecting 0-RTT at the TLS layer.  This also prevents
-QUIC from sending 0-RTT data. A server will always reject 0-RTT if it sends a
-TLS HelloRetryRequest.
+A server accepts 0-RTT by sending the EarlyDataIndication in the ServerHello
+(see Section 4.10.2 of {{!TLS13}}). A server that accepts 0-RTT MUST process and
+acknowledge all 0-RTT packets that it receives.
+
+A server rejects 0-RTT by sending a ServerHello with the EarlyDataIndication.
+When rejecting 0-RTT, a server MUST NOT process any 0-RTT packets, even if it is
+in possesion of the keys to do so. A server will always reject 0-RTT if it sends
+a TLS HelloRetryRequest.
```

Comment:
I think we should add text specifying what a client does if the server rejects 0-RTT but ACKs 0-RTT packets - probably PROTOCOL_VIOLATION.

Reply:
Thanks. I added a sentence about that.